ui/src/app/login/login/login.component.ts

```diff
@@ -67,23 +65,6 @@ export class LoginComponent implements OnInit {
         });
     }
 
-    getUserDevices() {
```

Comment:
Isn't there a button in the UI that triggers this function?

Reply:
no, but there is another one, we need to discuss what feature we want in the UI. Particularly the last page.